src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_administrated.xml

```diff
@@ -43,7 +43,7 @@
t;&lt;/f:function&gt;&lt;/body&gt;&lt;/html&gt;&lt;/f:param&gt;&lt;f:param name=&quot;RightPart&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;html&gt;&lt;head&gt;&lt;/head&gt;&lt;body&gt;&lt;f:function name=&quot;Composer.Banners.SimpleBanner&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;BannerImage&quot; value=&quot;MediaArchive:d0f35652-3fba-44da-9115-5e21634f7026&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;Alt&quot; value=&quot;Men Winter&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;Content&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;html&gt;&lt;head&gt;&lt;/head&gt;&lt;body&gt;&lt;/body&gt;&lt;/html&gt;&lt;/f:param&gt;&lt;f:param name=&quot;Actions&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;html&gt;&lt;head&gt;&lt;/head&gt;&lt;body&gt;&lt;f:function name=&quot;Composer.BootstrapButton&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;ButtonText&quot; value=&quot;Shop winter coats&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;ButtonSize&quot; value=&quot;normal&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;ButtonStyle&quot; value=&quot;secondary&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&lt;/body&gt;&lt;/html&gt;&lt;/f:param&gt;&lt;f:param name=&quot;ActionsHorizontalPos&quot; value=&quot;center&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;BackgroundColorRGB&quot; value=&quot;&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&lt;/body&gt;&lt;/html&gt;&lt;/f:param&gt;&lt;/f:function&gt;&lt;f:function name=&quot;Composer.SectionTitle&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;Title&quot; value=&quot;This week's specials&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;SubTitle&quot; value=&quot;&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&lt;f:function name=&quot;Composer.Queries.ProductSet.Top&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;QueryName&quot; value=&quot;FeaturedProducts&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;Number&quot; value=&quot;4&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&#xA;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" ChangeDate="2022-09-15 16:12:21Z" ChangedBy="admin" CreationDate="2020-04-28 14:38:50Z" CreatedBy="admin" PublicationStatus="published" SourceCultureName="en-CA" VersionId="94578974-decf-4222-8185-5b03d81cbc39" />
```

Comment:
this link is broken
![image](https://user-images.githubusercontent.com/6649972/190962739-70a105c6-ed01-4d80-be7c-3d15bac87198.png)

Reply:
commited the fix